### PR TITLE
feat(dedicated.ip): allow parking ip as destination

### PR DESCRIPTION
**Pull request by sachinrameshn**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-14569 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

Allow Parking as destination while ordering an additional ip.

### CODE REVIEW

**Review by JacquesLarique — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.constant.js [hunk 1]
```diff
@@ -67,6 +71,42 @@ export const BLOCK_ADDITIONAL_IP = 'Block Additional IP';
 
 export { TRACKING_PREFIX, ADDITIONAL_IP, ALERT_ID, IP_FAILOVER_PLANCODE };
 
+export const REGION_TO_COUNTRY = {
```
Comment (qpavy) [inline packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.constant.js:74]:
```suggestion
export const DATACENTER_TO_COUNTRY = {
```

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.constant.js [hunk 2]
```diff
@@ -77,4 +117,6 @@ export default {
   IP_FAILOVER_PLANCODE,
   BLOCK_ADDITIONAL_IP,
   ALERT_ID,
+  REGION_TO_COUNTRY,
```
Comment (qpavy) [inline packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.constant.js:120]:
```suggestion
  DATACENTER_TO_COUNTRY,
```

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js [hunk 3]
```diff
@@ -21,6 +21,7 @@ import {
   BLOCK_ADDITIONAL_IP,
   IP_FAILOVER_PLANCODE,
   ALERT_ID,
+  REGION_TO_COUNTRY,
```
Comment (qpavy) [inline packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js:24]:
```suggestion
  DATACENTER_TO_COUNTRY,
```

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js [hunk 4]
```diff
@@ -68,12 +69,7 @@ export default class AgoraIpV4OrderController {
     this.user = this.$state.params.user;
     this.catalogName = this.$state.params.catalogName;
     this.loadServices();
-  }
-
-  getIpv4Catalog() {
-    return this.ipCatalog.filter(
-      (plan) => plan.planCode.match(/^ip-v4.*|ip-failover.*/) != null,
-    );
+    this.ipv4Catalog = this.getIpv4Catalog();
```
Comment (qpavy) [inline packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js:72]: Do you need this line ? I don't see ipv4Catalog used

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js [hunk 5]
```diff
@@ -276,6 +288,9 @@ export default class AgoraIpV4OrderController {
     this.model.params = {};
     let ipOffersPromise;
 
+    this.isParkingIp =
+      get(this.model, 'selectedService.type') ===
+      PRODUCT_TYPES.parking.typeName;
```
Comment (qpavy) [inline packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js:293]:
```suggestion
    this.isParkingIp = this.model?.selectedService?.type === PRODUCT_TYPES.parking.typeName;
```

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js (hunk 4 above)
Comment (sachinrameshn) [reply packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js:72]: Its needed for second commit.

**Comment by sachinrameshn** [comment]
Taken care in another Pr.

**Pull request closed without merging**